google/resource_google_folder.go

```diff
@@ -0,0 +1,160 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	resourceManagerV2Beta1 "google.golang.org/api/cloudresourcemanager/v2beta1"
+	"strings"
+)
+
+func resourceGoogleFolder() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGoogleFolderCreate,
+		Read:   resourceGoogleFolderRead,
+		Update: resourceGoogleFolderUpdate,
+		Delete: resourceGoogleFolderDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceGoogleFolderImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			// Format is 'folders/{folder_id}.
+			// The folder id holds the same value.
```

Comment:
I think you mean 'id' or 'folder_id' as in terraform id, right? This comment was confusing to me because I don't see folder_id anywhere (this is exacerbated by the fact that other resources have fields like instance_id (which I am responsible for sorry) that are distinct from id).

Reply:
Replaced 'folder id' by 'terraform id'